libr/util/file.c

```diff
@@ -365,6 +365,9 @@ R_API char *r_file_slurp(const char *str, R_NULLABLE size_t *usz) {
 				}
 				size += r;
 			} while (!feof (fd));
+			if (!size) {
```

Comment:
probably there should always be `buf[size] = '\0';`, no matter what's the value of size.